### ISSUE TITLE
artifactregistry: missing fields in `artifactregistrypb.Registry`

### ISSUE THREAD

**Issue by antgamdia** [googleapis/google-cloud-go#8064]
**Client**

Importing the project and using the grpc client for making requests.

**Environment**

Local development, N/A

**Go Environment**

$ go version: go version go1.20.5 linux/amd64
$ go env N/A

**Code**

Related to this struct:

https://github.com/googleapis/google-cloud-go/blob/c7e8fd1c0d15c595d01fc354cdd0db709d6f9aaf/artifactregistry/apiv1/artifactregistrypb/repository.pb.go#L166-L198

**Expected behavior**

According to both [the documentation](https://cloud.google.com/artifact-registry/docs/reference/rpc/google.devtools.artifactregistry.v1#google.devtools.artifactregistry.v1.Repository) and the actual REST calls, the `Repository` data model returns some fields like `sizeBytes` or `mode`. For instance, this is a real call I have just performed:

```json
{
  "name": "projects/my-foo-project-id/locations/us-east1/repositories/my-cool-repo-id",
  "format": "DOCKER",
  "createTime": "2023-05-16T16:12:52.205145Z",
  "updateTime": "2023-05-16T16:12:52.205145Z",
  "mode": "STANDARD_REPOSITORY",
  "sizeBytes": "61885866"
}

```

**Actual behavior**

Some fields appearing in the [documentation](https://cloud.google.com/artifact-registry/docs/reference/rpc/google.devtools.artifactregistry.v1#google.devtools.artifactregistry.v1.Repository) are not being mapped to the GRPC message, therefore, they are not easily available when using the gcloud go SDK.

Specifically, I miss the following fields:

- `mode`
- `sizeBytes`
- `satisfiesPzs` (this is not being even returned in the REST API, but it is documented in the [docs site](https://cloud.google.com/artifact-registry/docs/reference/rpc/google.devtools.artifactregistry.v1#google.devtools.artifactregistry.v1.Repository))

**Additional context**

The problem applies to both `artifactregistry.apiv1beta2` and `artifactregistry.apiv1`.

**Comment by quartzmo** [comment]
@antgamdia Thank you for reporting this issue. It appears that the [repository.proto](https://github.com/googleapis/googleapis/blob/master/google/devtools/artifactregistry/v1/repository.proto) source file for the generated Go client has fallen out of sync with the documentation. I have opened an internal issue to bring this to the attention of the product team.

If your need to access these fields using Go is urgent, you can use the older, HTTP-based [google-api-go-client](https://github.com/googleapis/google-api-go-client/) library as the field appears to be published there: https://pkg.go.dev/google.golang.org/api/artifactregistry/v1#Repository. However, I believe the product team will respond quickly.

**Comment by noahdietz** [comment]
(I'm on rotation this week) Just bumped the internal issue to someone I think can help. Haven't forgotten about this!

**Comment by antgamdia** [comment]
Thanks for the quick response and the provided workaround. Since we already are consuming this SDK in our internal clients, we will probably want to wait until this issue gets fixed.

No rush, just for tracking it internally, would you have a rough ETA on when we can expect this to get solved?


Also, although this might be a separate issue (let me know if you want me to file it), we also found out some IAM admin operations are not available in the exposed IAMClient client but they are in the proto client. Perhaps it is the intended behavior, I don't know, but reporting it just in case :P 

--- 

- Example of an OK correspondence between the exposed IAMClient and the internal proto client:

https://github.com/googleapis/google-cloud-go/blob/54218e963bb5a6d47411a490985b54053825064f/iam/admin/apiv1/iam_client.go#L185

https://github.com/googleapis/google-cloud-go/blob/54218e963bb5a6d47411a490985b54053825064f/iam/admin/apiv1/adminpb/iam.pb.go#L5661


- Example of a NOK correspondence between the clients:

https://github.com/googleapis/google-cloud-go/blob/54218e963bb5a6d47411a490985b54053825064f/iam/admin/apiv1/adminpb/iam.pb.go#L5724

**Comment by noahdietz** [comment]
> although this might be a separate issue (let me know if you want me to file it),

Yeah I think this is :) Can you file another? I'll take a look at it. You can just copy the same comment into another issue.
